### PR TITLE
Fix for when the value doesnt get changed by the backend when we send it

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/more-info/controls/more-info-light.ts [hunk 1]
```diff
@@ -373,6 +369,8 @@ class MoreInfoLight extends LitElement {
       return;
     }
 
+    this._wvSliderValue = wv;
```
Comment (ludeeus) [inline src/dialogs/more-info/controls/more-info-light.ts:372]: Shouldn't this be set after `wv = Math.min(255, Math.round((wv * 255) / 100));` ?

**Review by ludeeus — APPROVED** [review]

File: src/dialogs/more-info/controls/more-info-light.ts (hunk 1 above)
Comment (bramkragten) [reply src/dialogs/more-info/controls/more-info-light.ts:372]: No, the slider is 0-100, the value we need to sent with the service call is 0-255

Comment (bramkragten) [reply src/dialogs/more-info/controls/more-info-light.ts:372]: But this is wrong... so thanks for pointing me to that 😄

File: src/dialogs/more-info/controls/more-info-light.ts [hunk 2]
```diff
@@ -369,7 +369,13 @@ class MoreInfoLight extends LitElement {
...
     }
 
-    this._wvSliderValue = wv;
+    if (name === "wv") {
+      this._wvSliderValue = wv;
+    } else if (name === "cw") {
+      this._cwSliderValue = wv;
+    } else if (name === "ww") {
+      this._wwSliderValue = wv;
+    }
```
Comment (ludeeus) [inline src/dialogs/more-info/controls/more-info-light.ts:378]:
```suggestion
	if (["wv", "cw", "ww"].includes(name)) {
      this._wwSliderValue = wv;
	}
```

Comment (bramkragten) [reply src/dialogs/more-info/controls/more-info-light.ts:378]: No that is not the same, you could do:

```js
this[`_${name}sliderValue`] = wv;

Comment (ludeeus) [reply src/dialogs/more-info/controls/more-info-light.ts:378]: great...! I read the `this._*` parts as identical :coffee:

**Pull request merged**